doc/source/rllib/package_ref/policy.rst

```diff
@@ -0,0 +1,32 @@
+.. _policy-reference-docs:
+
+Policy APIs
+===========
+
+.. toctree::
+   :maxdepth: 1
+
+   policy/policy.rst
+   policy/tf_policies.rst
+   policy/torch_policy.rst
+   policy/custom_policies.rst
+
+
+The ``Policy`` class contains functionality to compute actions for decision making
+in an environment, as well as computing loss(es) and gradients, updating a neural
+network model as well as postprocessing a collected environment trajectory.
+One or more ``Policy`` objects sit inside a
+`RolloutWorker's <https://github.com/ray-project/ray/blob/master/rllib/evaluation/rollout_worker.py>`_
```

Comment:
If we expect the user to know the rollout worker, can you make sure there is a reference to the API?

Reply:
ok, done